udp_ip: "10.42.0.1"
udp_port: 20000
cam_sn: "838212074315"
fx: 954.318
fy: 954.318
u:  961.893
v:  534.266